src/components/welcome-page/index.html

```diff
@@ -0,0 +1,31 @@
+<!--
+  Templates are written in HTML. We use a Handlebars-inspired syntax. Learn more
+  about the template syntax at
+  https://derbyjs.com/docs/derby-0.10/views/template-syntax
+-->
+
+<index:>
+  <div class="col-6 mx-auto">
+    <h2>
+      Welcome to the Derby Tutorial!
+    </h2>
+
+    <p>
+      Here are parts of the tutorial you can check out (recommended to go in order)
+    </p>
+
+    <hr/>
+    
+    <!-- https://derbyjs.com/docs/derby-0.10/views/template-syntax/blocks#conditionals -->
+    <ol>
+        <li><a href="/hello-world">Hello World</a></li>
+        <li><a href="/racer-basics">Racer Basics</a></li>
+        <li><a href="/components-in-action-demo">Components in Action Demo</a></li>
+        <li><a href="/collaborative-note-editor">Collaborative Note Editor</a></li>
+        <li><a href="/persistence-demo">Persistence Demo</a></li>
+    </ol>
+
+    <!--
+      Great! Now let's also take a look at the controller logic in `./index.js`
+    -->
+  </div>
```

Comment:
Let's add the trailing newline here - I believe it can be configured in VSCode with the "Files: Insert Final Newline" setting